druid/examples/multiwin.rs

```diff
@@ -78,7 +80,58 @@ fn ui_builder() -> impl Widget<State> {
     row.add_child(Padding::new(5.0, inc_button));
     row.add_child(Padding::new(5.0, dec_button));
     col.add_flex_child(Align::centered(row), 1.0);
-    col
+    Glow::new(col)
+}
+
+struct Glow<W> {
+    inner: WidgetPod<State, W>,
+}
+
+impl<W: Widget<State>> Glow<W> {
+    pub fn new(inner: W) -> Glow<W> {
+        Glow {
+            inner: WidgetPod::new(inner),
+        }
+    }
+}
+
+impl<W: Widget<State>> Widget<State> for Glow<W> {
+    fn event(&mut self, ctx: &mut EventCtx, event: &Event, data: &mut State, env: &Env) {
+        self.inner.event(ctx, event, data, env);
+    }
+
+    fn lifecycle(&mut self, ctx: &mut LifeCycleCtx, event: &LifeCycle, data: &State, env: &Env) {
+        if matches!(event, LifeCycle::HotChanged(_)) {
+            ctx.request_paint();
+        }
```

Comment:
For consistency with other examples
```suggestion
        if let LifeCycle::HotChanged(_) = event {
            ctx.request_paint();
        }
```